src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1376,6 +1376,10 @@ internal enum ErrorCode
 
         ERR_TupleElementNamesAttributeMissing = 8216,
         ERR_ExplicitTupleElementNames = 8217,
+        ERR_CantChangeTupleNamesOnOverride = 8218,
+        ERR_DuplicateInterfaceWithTupleNamesInBaseList = 8219,
+        ERR_ImplBadTupleNames = 8220,
+        ERR_PartialMethodTupleNameDifference = 8221,
 
         #endregion tuple diagnostics introduced in C# 7
```

Comment:
Range in `gap` comment below is now incorrect. Perhaps remove the range from the comment.